api/Makefile

```diff
@@ -1,3 +1,34 @@
+# Copyright The SPIFFE Project & Scytale, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# 	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
```suggestion
# limitations under the License.
#
# Modifications:
# <add a list of modifcations>
```

Not mandatory but friendly to the reader: add a short list of things you changed in this file